README.md

```diff
@@ -35,7 +33,7 @@ const server = new GraphQLServer({
     db: new Prisma({
       typeDefs: 'src/generated/prisma.graphql',
       endpoint: "__PRISMA_ENDPOINT__",
-      secret: 'mysecret123',
```

Comment:
if we keep the `secret` in `prisma.yml`, we also will need it in the constructor of the `GraphQLServer`